### ISSUE TITLE
[TP]-font injection triggers on change

### ISSUE THREAD

**Issue by NikolaRusakov** [NikolaRusakov/rusakov_website#7]
# Bug report

**What is the current behavior?**
On typography property change, injectFonts triggers.

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
InjectFonts would be triggered only in the following manner:
```
const injectRecentFont = useMemo(() => injectFonts(curTheme), [
    curTheme?.options?.headerFontFamily,
    curTheme?.options?.bodyFontFamily,
  ]);
```

**Other relevant information:**
- nullable typography on SSR, SSG process needs to be handled properly.
- Either inside project or library itself (injectFonts function)

Node.js version: 
NPM/Yarn version
Operating System: 
Additional tools:

**Comment by NikolaRusakov** [comment]
Fixed by #8

**Comment by NikolaRusakov** [comment]
Fixed by #8
